### PR TITLE
Scripts to deploy debian artifacts to Bintray

### PR DESCRIPTION
**Pull request by OYTIS**
Still under test

### CODE REVIEW

**Comment by OYTIS** [comment]
Looks good now.

File: travis/build-deb.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+if [[ ${TRAVIS_TAG} == v* ]]; then
+	VERSION=${TRAVIS_TAG:1}
+else
+	echo "Not in a tag, exit"
+	exit 1
+fi
+
+mkdir -p debs_to_deploy
+git clone https://github.com/oytis/raspbian-deb-builder
```
Comment (rbeiter) [inline travis/build-deb.sh:18]: Lets get this into a twilio owned repo before merging, we can reach out to @dkundel for review to set public today.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- sudo apt-get update
+script:
+- if [ -n "$TRAVIS_TAG" ]; then travis_wait 60 sudo ./travis/build-deb.sh; fi
+deploy:
+- provider: bintray
+  file: deploy-lib.json
+  skip_cleanup: true
+  on:
+    tags: true
+  user: agerasimov@twilio
```
Comment (rbeiter) [inline .travis.yml:21]: Can this be parameterized like the `BINTRAY_TOKEN`?

**Review by rbeiter — APPROVED** [review]
This LGTM! :+1:

**Pull request merged**